src/pages/gettingStarted.ts

```diff
@@ -0,0 +1,121 @@
+export const GETTING_STARTED_HTML = `<!DOCTYPE html>
+    <html>
+        <head>
+            <title></title>
+        </head>
+        <style>
+            .inv {
+                display: none;
+            }
+            .deviceSelector {
+                width: 250px;
+                border: 1px solid #3D484C;
+                margin: 0 0 5px;
+                padding: 8px;
+                border-radius: 5px;
+                font-size: 12px;
+                padding-right: 30px;
+            }
+            .codeBox {
+                display: block;
+                width: 90%;
+                margin: 10px;
+                padding: 15px;
+                text-align: left;
+                background: none;
+                border: 1px solid grey;
+                border-radius:4px;
+            }
+
+        </style>
+        <body>
+            <h1>Getting started</h1>
+
+            <select id="target" class="deviceSelector">
+                <option selected disabled>Select a Device</option>
+                <option value="micro:bit">micro:bit</option>
+                <option value="CPX">CPX</option>
+            <select>
+    
+            <div id="micro:bit" class="visibleElement">
+                <h2> Tutorial for micro:bit </h2>
+                <h3> 1. Import the micro:bit library to use it! (This is required)</h3>
+                <span class="codeBox">
+                    <pre>from microbit import *</pre>
+                </span>
+                <h3> 2. Light up your micro:bit with love by showing a heart.</h3>
+                <span class="codeBox">
+                    <pre>display.show(Image.HEART)</pre>
+                </span>                
+                <h3> 3. Use your micro:bit to tell the world how you’re feeling.
```

Comment:
It seems you use an ordered list here, but you don't necessarily need to do all steps in order (E.g. You could do step 1 followed by step 4). From all the steps only the first one seems to be required. Maybe play around with the layout so it reflects that?

Reply:
Makes sense, we could make a separate category for importing

Reply:
Will be fixed in future iterations